management-ui/static/m1-user-interface.js

```diff
@@ -9,6 +9,8 @@ https://drive.google.com/file/d/1cinCiA778IErENZ3JN52VFW-1ffHpx7Z/view
 
 // Auxiliary function to clear storage when the connection is lost
 
+let operatingUrl = 'https://5gms-provisioning-portal.5g-mag.com/webui';
```

Comment:
If you default this to `''` and use `${operatingUrl}rest_of_path` (e.g. `${operatingUrl}connection_checker`, note no `/` in the middle) in your format strings then you'll get relative URLs which will work with any path for the nginx reverse-proxy. This could then be overridden on an installation with a string like `'https://my.example.com/5gms-ui/'` if needed for a given installation (maybe leave a comment in the code to that effect).

Reply:
#46 merged to `development`, ready for this pull to `master`.